README.md

```diff
@@ -196,9 +197,9 @@ Visit postman to checkout the collection of enpoints and try it locally.
 
     Deletes an individual menu by id
 
-##### GET "/menus/:id/menus"
+##### GET "/menus/:id/recipes"
 
```

Comment:
We do not have this get method, instead, we created a DELETE crud, could you please update that? 

DELETE "/menus/:id/recipeId"
Deletes the the menu recipe relationship in the menu recipe join table